apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -43,18 +41,17 @@ public final class RequestResponseLoggingFilter implements Filter {
   @Override
   public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
       throws IOException, ServletException {
+    /*
+     * This should be cleared by Jetty via the request log handling but just in case we have a situation where
+     * that handler does not fire (say, due to a Jetty defect) clear it now before the request starts.
+     */
+    MDC.clear();
+
+    handleRequest(request);
     try {
-      handleRequest(request);
```

Comment:
Is this not needed inside the try anymore? It seems to be the same call.

Reply:
the outer try was just to have the finally block that clears MDC. The MDC clearing is now done in DataServerLauncherApp.java and so the finally block was moved to that new location.